packages/rancher-aks-operator-crd/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/aks-operator/releases/download/v1.0.2/rancher-aks-operator-crd-1.0.2.tgz
+url: https://github.com/rancher/aks-operator/releases/download/v1.0.3-rc1/rancher-aks-operator-crd-1.0.3-rc1.tgz
 version: 100.0.1
```

Comment:
I think this version should be bumped as well to stay inline with the aks-operator chart.

Reply:
that version is typically for the changes specific to the charts repo, if we bump upstream it will be added as  tag to the semver.